packages/artifacts/chunks/languages/builtins.txt

```diff
@@ -0,0 +1,18 @@
+language LionCore-builtins
+    version: 2023.1
+    entities (↓name):
+
+        primitive type Boolean
+
+        concept-interface INamed
+            features (↓name):
```

Comment:
What is the arrow supposed to mean?

Reply:
It's an indication that the features are sorted by name.